### PR TITLE
Update changelog with 0.2 updates.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Merge in the unreleased 0.1.5 version changelog.

I'm not sure there's much point in trying to be exhaustive here, there's a lot of changes since 0.1.4 ~2 years ago.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,67 +1,66 @@
...
-[0.2.0] - XXXX-XX-XX
+[0.2.0] - 2020-12-18
 ********************
 
+Major feature release, including some incompatible file format and API updates.
+
 **New features**:
 
+- Mismatch and recombination parameters can now be specified via the
+  recombination_rate and mismatch arguments in the Python API.
```
Comment (hyanwong) [inline CHANGELOG.rst:10]: `mismatch_ratio` I think (we are hiding the `mismatch` parameter

**Review by hyanwong — APPROVED** [review]
All LGTM. I agree about not being exhaustive.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,67 +1,66 @@
...
+
+- Missing data can be accomodated in SampleData using the tskit.MISSING_DATA
+  value in input genotypes.
```
Comment (hyanwong) [inline CHANGELOG.rst:13]: Probably worth saying that missing data will be imputed in the final TS.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,67 +1,66 @@
...
+
+- Metadata schemas for population, individual, site and tree sequence metadata
+  can now we be specified in the SampleData format. These will be included
+  in the final tree sequence and allow for automatic decoding of JSON metadata.
```
Comment (hyanwong) [inline CHANGELOG.rst:17]: Do we want to say that we add metadata to sites & individuals when doing inference? I guess it's not that important for most people

**Comment by hyanwong** [comment]
Oh, one last thing: we added the "proxy ancestors" stuff in there to help support historical samples, along with e.g. `match_samples(..., force_sample_times=True)` which is quite a big thing, but is a little bit bound up with tsdate. Possibly worth mentioning though. Do you want me to write a sentence?

File: CHANGELOG.rst (hunk 3 above)
Comment (jeromekelleher) [reply CHANGELOG.rst:17]: I think that's the role of documentation, this is just to let people know what changed.

**Comment by jeromekelleher** [comment]
Can you give me a sentence for the proxy ancestors thing here please @hyanwong? I think this basically wraps it up now, I'm aiming to merge this and tag the release now.

**Comment by hyanwong** [comment]
How's this @jeromekelleher ?

```
- Historical (non-contemporaneous) samples can now be accommodated in inference,
   assuming that the true dates of ancestors have been set, by using the concept of "proxy samples".
   This is done via the new function ``AncestorData.insert_proxy_samples()``, then setting the new
   parameter ``force_sample_times=True`` when matching samples.
```

**Pull request merged**